frontend/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@bailo/frontend",
   "private": true,
   "scripts": {
-    "dev": "next dev -- --turbo",
+    "dev": "next dev",
```

Comment:
Should this change be in this PR?

Reply:
Yeah, it doesn't seem to work with several aspects of our application.